python_cycles/project_cycles.py

```diff
@@ -0,0 +1,84 @@
+"""Проект з циклами
+    Генератор числових шаблонів
+Мета: Створити програму, яка генерує різні шаблони за допомогою різних типів циклів.
+    Функції:
+Шаблони трикутників з чисел
+Генерація таблиці множення
+Послідовність зворотного відліку
+    Фокус реалізації:
+Цикли for з range() для фіксованих ітерацій
+Вкладені цикли для генерації 2D-шаблонів
+Цикли while для послідовностей зворотного відліку
+Демонстрація структури керування циклами
+    Інтерфейс користувача: 
+Інтерфейс на основі меню для вибору типів шаблонів та параметрів.
+    Виклики:
+Створення різних стилів шаблонів за допомогою циклів
+Реалізація вкладених циклів для складних шаблонів
+Акуратне форматування виводу"""
+
+
+print("\nВітаємо Вас у Генераторі числових шаблонів")
+
+
+while True:  # Створюємо цикл, який буде працювати, поки виконуються умови циклу.
+    try:
+        # Запит на вибір функції
+        check_func = int(input("\nОберіть будь ласка номер функції: \n"
+                       "\n1 - Шаблони трикутників з чисел\n"
+                       "2 - Генерація таблиці множення\n"
+                       "3 - Послідовність зворотного відліку\n"
+                       "4 - Вихід\n"
+                        "\nВведить обраний номер функції: "))
```

Comment:
Use proper `ruff-style` indentation:
```suggestion
        check_func = int(
            input(
                "\nОберіть будь ласка номер функції: \n"
                "\n1 - Шаблони трикутників з чисел\n"
                "2 - Генерація таблиці множення\n"
                "3 - Послідовність зворотного відліку\n"
                "4 - Вихід\n"
                "\nВведить обраний номер функції: ",
            ),
        )
```